projects/kodiak-v3/index.js

```diff
@@ -8,7 +8,13 @@ module.exports = {
     berachain: {
       factory,
       fromBlock,
-      permitFailure: true
+      permitFailure: true,
+      blacklistedTokens: [
```

Comment:
this uses `sumTokens2` underneath, and it has a `blacklistedOwners` field, so you can use it to blacklist pools instead of tokens